t/watcher.t

```diff
@@ -235,3 +235,31 @@ SKIP: {
 
     is($watchresult, "CLIENT_ERROR watch commands not allowed\r\n", "attempted watch gave client error with no_watch option set");
 }
+
+
+# test delete/meta-delete with value sizes
+{
+    my $watcher = $server->new_sock;
```

Comment:
Brings a tear to my eye to see tests in an outside PR!

If the PR's intention is that it only logs on delete hit, there should be tests specifically for that as well.

Reply:
haha - yes. will add